### PR TITLE
chore(sweeps): add warning for non-custom methods with scheduler jobs

### PR DESCRIPTION
**Pull request by gtarpenning**
Description
-----------
Anaconda2 needs a method, so for scheduler jobs using anaconda we need a method.

### CODE REVIEW

File: wandb/cli/cli.py [hunk 1]
```diff
@@ -977,10 +977,10 @@ def launch_sweep(
         if scheduler_job and not method:
             sweep_config["method"] = "custom"
         elif scheduler_job and method != "custom":
-            wandb.termerror(
-                "Using a scheduler job for launch sweeps is only supported with the 'custom' method"
+            wandb.termwarn(
+                "Using a scheduler job for launch sweeps is only supported with the 'custom' method, "
+                "except for scheduler jobs inheriting from the WandbScheduler class"
```
Comment (KyleGoyette) [inline wandb/cli/cli.py:982]: This isn't very actionable I think, would users know what the WandbScheduler is?

**Review by KyleGoyette — APPROVED** [review]
I'd like to see a long term solution for being able to tell whether the method should be custom or not and fixing it with a warning, instead of a best effort fix that we see here.

Might also want to have a technical FAQ in the SoL docs, and point to that here

**Comment by gtarpenning** [comment]
> /fix-title

LOL it changed `sweeps` to `sweaps`

**Comment by dmitryduev** [comment]
not cool, GPT! 🤣

**Pull request merged**